design/environment.go

```diff
@@ -25,14 +24,15 @@ var envAttrs = a.Type("EnvironmentAttributes", func() {
 		a.Example("myapp-stage")
 	})
 	a.Attribute("type", d.String, "The environment type", func() {
-		a.Example("stage")
+		a.Enum("dev", "build", "stage", "run")
 	})
 	a.Attribute("namespaceName", d.String, "The namespace name", func() {
 		a.Example("myapp-stage")
 	})
 	a.Attribute("cluster-url", d.String, "The cluster url", func() {
 		a.Example("https://api.starter-us-east-2a.openshift.com")
 	})
```

Comment:
@alexeykazakov do you see `cluster-url` also should be mandatory ?? and we should make it `NOT NULL`

Reply:
Done [6da35e6](https://github.com/fabric8-services/fabric8-env/pull/26/commits/6c8f2101bd7a2cf2ccc0155a8e914f6fa6da35e6)